test/e2e_node/runner/remote/run_remote.go

```diff
@@ -119,11 +129,12 @@ type internalImageConfig struct {
 }
 
 type internalGCEImage struct {
```

Comment:
Should we more formally organize these types by splitting into separate internal/external files?